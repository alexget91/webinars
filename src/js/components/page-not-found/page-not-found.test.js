import React from "react";
import renderer from "react-test-renderer";
import PageNotFound from "./page-not-found";

it(`Page not found message correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<PageNotFound/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
