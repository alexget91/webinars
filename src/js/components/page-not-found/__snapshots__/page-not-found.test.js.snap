// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page not found message correctly renders after relaunch 1`] = `"Page not found!"`;
